data/appdata-xml.m4

```diff
@@ -36,7 +36,7 @@ AU_DEFUN([APPDATA_XML],
          AS_IF([test "x$enable_appdata_validate" = "xyes"],
                [AC_MSG_ERROR([AppData validation was requested but appstream-util was not found])])])
 
-  AC_SUBST([appdataxmldir], [${datadir}/appdata])
+  AC_SUBST([appdataxmldir], [${datadir}/metainfo])
```

Comment:
We have a couple of them on Debian, and this is slowly making its way upstream too (I would have switched KDE already, but we need to be sure all distros support it before making this change).
Scribus in Debian and Ubuntu would be an example: https://packages.debian.org/experimental/scribus-ng
(I also know at least three other packages, but I am not sure how helpful Debian packages are for you)
In any case, it's safe to assume that this will make its way upstream rather quickly.